.gitignore

```diff
@@ -120,3 +120,6 @@ __pycache__/
 
 # Idea
 .idea/
+
+# pnpm lock file
+pnpm-lock.yaml
```

Comment:
Why ignore this `pnpm-lock.yaml`? Think this is necessary.

Reply:
`pnpm-lock.yml` needs to be updated, after `pnpm install` this file is changed.